another-redis-desktop-manager@1.7.0: download via aria2 failed

```powershell
Installing 'another-redis-desktop-manager' (1.7.0) [64bit] from 'extras' bucket
Starting download with aria2 ...
Download: 11/21 12:01:27 [ERROR] CUID#8 - Download aborted. URI=https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v1.7.0/Another-Redis-Desktop-ManagDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v1.7.0/Another-Redis-Desktop-ManagerDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: ef845f|ERR |       0B/s|C:/Users/xxx/scoop/cache/another-redis-desktop-manager#1.7.0#c6c92bb.7z
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v1.7.0/Another-Redis-Desktop-Manager.1.7.0.exe#/dl.7z
    referer=https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v1.7.0/Another-Redis-Desktop-Manager.1.7.0.exe#/
    dir=C:\Users\xxx\scoop\cache
    out=another-redis-desktop-manager#1.7.0#c6c92bb.7z

ERROR & 'C:\Users\xxx\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\xxx\scoop\cache\another-redis-desktop-manager.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.4 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=24556 --continue --summary-interval=0 --auto-save-interval=1
```

--- a/bucket/another-redis-desktop-manager.json
+++ b/bucket/another-redis-desktop-manager.json
@@ -5,7 +5,7 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v1.7.0/Another-Redis-Desktop-Manager.1.7.0.exe#/dl.7z",
+            "url": "https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v1.7.0/Another-Redis-Desktop-Manager-win-1.7.0-x64.exe#/dl.7z",
             "hash": "sha512:18b59199956c583bf4f96595dfa1bd40d5c5a827db1ee57bbf22f84e67479e355710987f9db2852016e584d4945aae6f94f3271b150308cd200ee492bae378f8"
         }
     },
@@ -23,7 +23,7 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v$version/Another-Redis-Desktop-Manager.$version.exe#/dl.7z",
+                "url": "https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v$version/Another-Redis-Desktop-Manager-win-$version-x64.exe#/dl.7z",
                 "hash": {
                     "url": "$baseurl/latest.yml",
                     "regex": "sha512:\\s+$base64"

